domain/domain.go

```diff
@@ -118,6 +118,14 @@ type Domain struct {
 	sysExecutorFactory   func(*Domain) (pools.Resource, error)
 
 	sysProcesses SysProcesses
+
+	mdlCheckTableInfo *mdlCheckTableInfo
+}
+
+type mdlCheckTableInfo struct {
+	mu         sync.Mutex
```

Comment:
I saw you add lock to avoid concurrent write jobsVerMap and jobsIDsMap,  But if like your before implement. only mdlCheckLoop will change them, seems we do not need to split it to two coroutine?

Reply:
So that we don't need to do the `select` every time.